docs/Using-Fleet/Teams.md

```diff
@@ -50,6 +50,43 @@ To create a team:
 
 3. Enter your new team's name and select "Save."
 
+## Apply agent options to a team
+
+You can set up team-specific agent options. If you set agent options for your team, any host enrolled to that team will be configured with the team's agent options instead of your organization's [Global agent options](./Fleet-UI.md#global-agent-options). Make sure you include any global agent options that you also want applied to your team. 
+
+Example team agent options:
+
+```YAML
+config:
+  decorators:
+    load:
+      - SELECT uuid AS host_uuid FROM system_info;
+      - SELECT hostname AS hostname FROM system_info;
+  options:
+    disable_distributed: false
+    distributed_interval: 10
+    distributed_plugin: tls
+    distributed_tls_max_attempts: 3
+    logger_plugin: tls
+    logger_tls_endpoint: /api/v1/osquery/log
+    logger_tls_period: 10
+    pack_delimiter: /
+  overrides: {}
+```
+
+To set agent options for a team:
+
+1. In the top navigation select "Settings" and then, in the sub-navigation, select "Teams."
+
+2. Find and select your team. 
+
+3. Select the "Agent options" tab. 
+
+4. Edit the options in the `YAML` editor. 
+
+5. Click "Save options" on the right.
```

Comment:
I wonder if there's more here. OK, I've generated this YAML, now what? 

I thought you have to re-generate the installers and ship them to your end users, right? Maybe discuss that and/or link to the relevant article that tells you how to do that? Or do I change the YAML here and then do fleetctl apply? If so, should discuss that or link to the relevant area.